src/main/scala/com/softwaremill/mqperf/mq/KafkaMq.scala

```diff
@@ -1,31 +1,32 @@
-package com.softwaremill.mqperf.mq.kafka
+package com.softwaremill.mqperf.mq
+
+import java.util.{Properties, Map => JMap}
 
-import java.util
-import java.util.{Properties, UUID}
-import akka.actor.{ActorSystem, Props}
-import com.softwaremill.mqperf.mq.Mq
 import com.softwaremill.mqperf.{ReceiverRunnable, ReportResults}
 import com.typesafe.scalalogging.StrictLogging
-import org.apache.kafka.clients.consumer.{ConsumerRecord, KafkaConsumer}
+import org.apache.kafka.clients.consumer.{ConsumerRecord, KafkaConsumer, OffsetAndMetadata, OffsetCommitCallback}
 import org.apache.kafka.clients.producer._
 import org.apache.kafka.common.TopicPartition
-import scala.collection.JavaConversions._
+
 import scala.annotation.tailrec
-import scala.concurrent.Await
+import scala.collection.JavaConversions._
 import scala.concurrent.duration._
 import scala.language.postfixOps
 import scala.util.Random
 
 class KafkaMq(configMap: Map[String, String]) extends Mq with StrictLogging {
-  private val GroupId = "mq-group"
-  private val Topic = "mq"
-  val pollTimeout = 50 millis
-  lazy val system = ActorSystem(s"KafkaTest-${UUID.randomUUID().toString}")
+  import KafkaMq._
+
+  private val GroupId = "mq-group2"
+  private val Topic = "mq2"
+  val pollTimeout = 50.millis.toMillis
+  var offsetsToCommit: CommitOffsets = Map.empty
```

Comment:
Good catch! `offsetsToCommit` and `lastCommitTick` should be indeed fields of a `Receiver`, not `KafkaMq`. Fixing :)